test/services/trails_service_test.rb

```diff
@@ -9,22 +9,49 @@ def setup
 
   test "#trails" do
     VCR.use_cassette("trails_service_api") do
-      trails = service.trails
-      first_trail = trails.first
-
-      assert_equal "San Francisco", first_trail["city"]
-      assert_equal "California", first_trail["state"]
-      assert_equal "Mt. Sutro Historic Trail", first_trail["name"]
-      assert_equal 664, first_trail["unique_id"]
-      assert_equal 37.75756, first_trail["lat"]
-      assert_equal -122.45615, first_trail["lon"]
+      all_trails   = service.get_all_trails
+      trails       = all_trails.flatten
+      first_trail  = trails.first
+
+      assert_equal 51, trails.count
```

Comment:
I figured for the vcr tests it was best to set the limit to 1 per state. so i changed the limit to 1 prior to running the tests and setting up the vcr. It takes quite a bit of time to go through all of them.